esti/main_test.go

```diff
@@ -86,6 +292,15 @@ func TestMain(m *testing.M) {
 	logger, client, svc = testutil.SetupTestingEnv(&params)
 
 	var err error
+	if *cleanupEnv {
```

Comment:
I think it makes more sense to put the call to `envCleanup` in `SetupTestingEnv` inside the else block of `if setupLakeFS `

Reply:
1. to clean up my env, we create one only after this if-else statement.
2. If someone uses the setup lakeFS env, he shouldn't use the cleanup flag too, it doesn't relevant for him.

Reply:
So we should protect against using the ESTI_SETUP_LAKEFS env var and the cleanup flag. But I think it will be more logical to just put it in the setup flow